lib/get-s3-api.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+var _ = require('ls-lodash'),
+    aws = require('aws-sdk'),
+    s3Config = {};
```

Comment:
http://blogs.aws.amazon.com/security/post/Tx3D6U6WSFGOK2H/A-New-and-Standardized-Way-to-Manage-Credentials-in-the-AWS-SDKs